osu.Game/Skinning/LegacySkin.cs

```diff
@@ -364,6 +364,8 @@ protected override void ParseConfigurationStream(Stream stream)
                         case GlobalSkinnableContainers.MainHUDComponents:
                             if (containerLookup.Ruleset != null)
                             {
+                                // Notably, there may be existing ruleset skin transformers that override the default layout returned here.
+                                // Any added component in here should be considered in the existing transformers if applicable.
```

Comment:
Added comment because I tripped on this one and caused a total disaster (as seen above).